<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsKeyboard</class>
 <widget class="QWidget" name="SettingsKeyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="m_assignTable"/>
   </item>
   <item>
    <widget class="QCheckBox" name="m_appleIRRemote">
     <property name="text">
      <string>Enable control using Apple IR Remote</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_lircGroup">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>WinLIRC Remote</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="m_lircHostLabel">
          <property name="text">
           <string>WinLIRC Host</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_lircHostEdit">
          <property name="text">
           <string>localhost</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="m_lircPortLabel">
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="m_lircPortEdit">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>65536</number>
          </property>
          <property name="value">
           <number>8765</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="m_lircRecieveIcon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../player.qrc">:/winlirc/Resources/winlirc/remote_ok.png</pixmap>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="m_lircStatusIcon">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../player.qrc">:/player/Resources/exclamation.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="m_lircStatusText">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Not connected to WinLIRC Server</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>5</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_assignTable</tabstop>
  <tabstop>m_appleIRRemote</tabstop>
  <tabstop>m_lircGroup</tabstop>
  <tabstop>m_lircHostEdit</tabstop>
  <tabstop>m_lircPortEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../player.qrc"/>
 </resources>
 <connections/>
</ui>
